client/src/controllers/LoaderController.ts

```diff
@@ -0,0 +1,92 @@
+import { Controller } from '@hotwired/stimulus';
+
+/**
+ * <button
+ *   type="submit"
+ *   class="button button-longrunning"
+ *   data-controller="w-loader"
+ *   data-w-loader-active-class="button-longrunning-active"
+ *   data-w-loader-active-value="{% trans 'Signing in…' %}"
+ *   data-w-loader-duration-seconds-value="40"
+ *   data-action="w-loader#activate"
+ * >
+ *  {% icon name="spinner" %}
+ *  <em data-w-loader-target="label">{% trans 'Sign in' %}</em>
+ * </button>
+ */
+export class LoaderController extends Controller {
+  static classes = ['active'];
+  static targets = ['label'];
+  static values = {
+    active: { type: String, default: 'Loading' },
```

Comment:
```suggestion
    active: { type: String, default: 'Loading' },
```

we should be careful here when writing strings in JavaScript, remember to use the translation utils.

e.g.
```
import { gettext } from '../../utils/gettext';
gettext('Loading')
```